EnglishFilter/sections/cname_adservers.txt

```diff
@@ -1,5 +1,9 @@
 !
-! CNAME advertising networks
+! This section contains the list of the rules are against CNAME advertising networks.
+!
+! Good: !#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/trackers/adspyglass.txt
```

Comment:
It should be explained that we only include rules from AdGuard's cname-tracker repo here, there's no need to add regular rules here.

Reply:
Added a note about it.